src/LoadingPage/styled.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import { ReactComponent as LoadingIcon } from "../assets/loadingIcon.svg";
+
+export const Container = styled.div`
+  padding: 16px;
+`;
+
+export const StyledParagraph = styled.p`
+  font-weight: 600;
+  font-size: 36px;
+  line-height: 43.2px;
+  padding: 0 0 120px;
+  margin: 0;
```

Comment:
You can increase margin-left value to 276px on desktop for better visibility and compliance with design in Figma🤗

Reply:
Compliance with the project will be achieved by placing this component in <article>, which will be limited to 1400px in width and centered.